### ISSUE TITLE
RAF features not working correct

### ISSUE THREAD

**Issue by mohitmistry** [dalaranwow/dalaran-wow#4822]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
I quest with a friend who i used the RAF code for during the sign up. I timed out due to internet issues and then refreshed, checked the code, and resubmitted the sign up sheet. Will give code later if needed. When I group with said friend under the account, "Summon Linked Account" is shown in the Friends list. The EXP is also not working correctly under the friend as everything done under questing with them gives normal EXP. Please let me know if there was a backend issue and I will help as best as I can providing info.

**Expected behaviour**: Tell us what should happen instead
The EXP should work under the friend who summoned me.

**Steps to reproduce the problem**:

1. Play with friend?  
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
Refer-a-friend bonuses works fine, so if you have created the account through the mail you received it should be fine.

Requirements to receive XP bonus:
1) You must have your friend in your friend list, and vice versa.
2) You must be close to each other.
3) You must be in party with your friend.
4) You both must be within the same level range and less than level 60.

---

If the issue persists, you can always make an in-game ticket and we'll make sure to check if something went wrong. Make sure to provide both account names.

Thanks!

![GitHub Logo](http://i.imgur.com/bYkYeGQ.png)

See https://github.com/dalaranwow/dalaran-wow/issues/3325 for further information
